pkg/storage/mongo/saver.go

```diff
@@ -2,17 +2,23 @@ package mongo
 
 import (
 	"context"
+	"io"
+	"io/ioutil"
 
 	"github.com/gomods/athens/pkg/storage"
 )
 
 // Save stores a module in mongo storage.
-func (s *ModuleStore) Save(_ context.Context, module, version string, mod, zip, info []byte) error {
+func (s *ModuleStore) Save(_ context.Context, module, version string, mod []byte, zip io.Reader, info []byte) error {
+	z, err := ioutil.ReadAll(zip)
```

Comment:
This is fine as `z` but could probably be a bit more readable as `zipBytes` as elsewhere in the PR